enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -2594,24 +2594,27 @@ func (c *FirecrackerContainer) reclaimMemoryWithBalloon(ctx context.Context) err
 	ctx, span := tracing.StartSpan(ctx)
 	defer span.End()
 
+	// Drop the page cache to free up more memory for the balloon.
+	// The balloon will only allocate free memory.
 	conn, err := c.vmExecConn(ctx)
 	if err != nil {
 		return status.InternalErrorf("failed to dial VM exec port: %s", err)
 	}
 	client := vmxpb.NewExecClient(conn)
 	cmd := &repb.Command{
-		Arguments: []string{"sh", "-c", "free | awk '/^Mem:/ {print $7}'"},
+		Arguments: []string{"sudo", "sh", "-c", "echo 3 > /proc/sys/vm/drop_caches"},
```

Comment:
Doesn't have to be in this PR but we probably want a dedicated RPC to drop the page cache to avoid depending on `sudo` or `sh` being available in the user's container image.

(btw you can remove `sudo` if you pass `"0:0"` instead of `c.user` to the `Execute()` command)

Reply:
Ack - changed to use 0:0 and will remove `sh` in a follow up